spec/unit/pdk/generate/module_spec.rb

```diff
@@ -222,9 +222,11 @@
       context 'when a template-url is supplied on the command line' do
         before(:each) do
           allow(PDK::Util::Filesystem).to receive(:mv).with(temp_target_dir, target_dir).and_return(0)
-          allow(PDK::Util).to receive(:default_template_uri).and_return(Addressable::URI.parse('https://github.com/puppetlabs/pdk-templates'))
+          allow(PDK::Util).to receive(:default_template_uri).and_return(Addressable::URI.parse(default_template_url))
         end
 
+        let(:default_template_url) { 'https://github.com/puppetlabs/pdk-templates' }
```

Comment:
nit : Should this let be above the `before(:each)`.  I know ruby doesn't really care, but seems odd to use the variable first, and then define it later.

Reply:
updated